src/util/mega_editor.js

```diff
@@ -44,15 +42,11 @@ export const megaEdit = async function (postIds, options) {
     delete requestBody.tags;
   }
 
-  return inject(
-    async (resource, body) =>
-      fetch(resource, {
-        method: 'POST',
-        headers: { 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8' },
-        body
-      }).then(async response =>
-        response.ok ? response.json() : Promise.reject(await response.json())
-      ),
-    [`https://www.tumblr.com/${pathname}`, $.param(requestBody)]
+  await fetch(`https://www.tumblr.com/${pathname}`, {
```

Comment:
this should be a `return` for if we ever want to access the response or, more importantly, any errors

```suggestion
  return fetch(`https://www.tumblr.com/${pathname}`, {
```

Reply:
You get errors with await! But yeah, should be able to get the response too, huh.